From d067e3342fefe0be14ce5294a53ea2c76a2a0d04 Mon Sep 17 00:00:00 2001
From: antirez <antirez@gmail.com>
Date: Thu, 22 Jun 2017 11:00:21 +0200
Subject: [PATCH 19/39] Aesthetic changes to #4068 PR to conform to Redis
 coding standard.

1. Inline if ... statement if short.
2. No lines over 80 columns.
---
 src/aof.c | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/src/aof.c b/src/aof.c
index ea9ac47a..3b80ca1d 100644
--- a/src/aof.c
+++ b/src/aof.c
@@ -542,17 +542,16 @@ void feedAppendOnlyFile(struct redisCommand *cmd, int dictid, robj **argv, int a
         /* Translate SET [EX seconds][PX milliseconds] to SET and PEXPIREAT */
         buf = catAppendOnlyGenericCommand(buf,3,argv);
         for (i = 3; i < argc; i ++) {
-            if (!strcasecmp(argv[i]->ptr, "ex"))
-                exarg = argv[i+1];
-
-            if (!strcasecmp(argv[i]->ptr, "px"))
-                pxarg = argv[i+1];
+            if (!strcasecmp(argv[i]->ptr, "ex")) exarg = argv[i+1];
+            if (!strcasecmp(argv[i]->ptr, "px")) pxarg = argv[i+1];
         }
         serverAssert(!(exarg && pxarg));
         if (exarg)
-            buf = catAppendOnlyExpireAtCommand(buf,server.expireCommand,argv[1],exarg);
+            buf = catAppendOnlyExpireAtCommand(buf,server.expireCommand,argv[1],
+                                               exarg);
         if (pxarg)
-            buf = catAppendOnlyExpireAtCommand(buf,server.pexpireCommand,argv[1],pxarg);
+            buf = catAppendOnlyExpireAtCommand(buf,server.pexpireCommand,argv[1],
+                                               pxarg);
     } else {
         /* All the other commands don't need translation or need the
          * same translation already operated in the command vector
-- 
2.14.1

